Readme.md

```diff
@@ -3,49 +3,94 @@
 
-<p>This sample demonstrates how to implement a simple custom binding scenario for the GridView extension by handling only sorting and paging operations in the corresponding Action methods.</p>
-<p>To learn more on the GridView's custom data binding feature, please refer to the <a href="https://docs.devexpress.com/AspNetMvc/14321/components/grid-view/concepts/binding-to-data/custom-data-binding?p=netframework"><u>Custom Data Binding</u></a> help topic.</p>
-<br>
-<p>Note that this sample provides a universal implementation approach - it can be easily adopted and used for every data source object that implements the <strong>IQueryable</strong> interface.</p>
-<p> </p>
-<p>In short, the logic of this custom binding implementation is as follows:</p>
-<br>
-<p>In the GridView's Partial View (Views > Home > GridViewPartial.cshtml), the grid's <a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewSettings.CustomBindingRouteValuesCollection?p=netframework"><u>CustomBindingRouteValuesCollection</u></a> property is used to define handling actions for sorting and paging operations; the <a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GanttSettings.CallbackRouteValues?p=netframework"><u>CallbackRouteValues</u></a> property defines the action to handle all other (standard) grid callbacks.</p>
-<br>
-<p>In the Controller (Controller > HomeController.cs), the specified Action methods are implemented to update a specific grid view model object (<a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewModel?p=netframework"><u>GridViewModel</u></a> that maintains the grid state) with the information of the performed operation (if required). Then, the grid view model's <a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewModel.ProcessCustomBinding.overloads?p=netframework"><u>ProcessCustomBinding</u></a> method is called to delegate a binding implementation to specific model-layer methods pointed by the method's certain parameters.</p>
-<br>
-<p>At the Model layer (Models > CustomBindingModel.cs), the two specified delegates are implemented to populate the grid view mode with the required data. Generally, in the provided implementation of model-level binding delegates, you just need to modify a single code line to point to your particular model object:</p>
+You can modify this approach to use with any data source object that implements the `IQueryable` interface.
 
 
-```cs
-        static IQueryable Model { get { return NorthwindDataProvider.GetCustomers(); } }
+## Implementation details
 
+Custom data binding requires the [DevExpressEditorsBinder](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.DevExpressEditorsBinder) is used instead of the default model binder to correctly transfer values from DevExpress editors back to the corresponding data model fields. 
```

Comment:
```suggestion
Custom data binding requires that the [DevExpressEditorsBinder](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.DevExpressEditorsBinder) is used instead of the default model binder to correctly transfer values from DevExpress editors back to the corresponding data model fields. 
```